server/src/main/java/org/elasticsearch/bootstrap/Elasticsearch.java

```diff
@@ -200,7 +205,17 @@ private static void initPhase2(Bootstrap bootstrap) throws IOException {
         );
 
         if (Boolean.parseBoolean(System.getProperty("es.entitlements.enabled"))) {
-            EntitlementBootstrap.bootstrap();
+            Map<Path, Boolean> pluginData = new HashMap<>();
```

Comment:
Note: after https://github.com/elastic/elasticsearch/pull/116998, this will use `moduleDescriptors` and `pluginDescriptors`  from `PluginsLoader` - in this PR if this gets merged after https://github.com/elastic/elasticsearch/pull/116998, or in a separate follow-up if the merge order is inverted.